setup.cfg

```diff
@@ -9,6 +9,7 @@ branch = True
 
 [coverage:report]
 show_missing = True
+fail_under = 95
```

Comment:
For the record I think 95% is almost a bit too high - I like to aim for 90% so we don't hit diminishing returns, given code coverage is just an imprecise measurement. But given we're already this high I'm willing to see whether we can keep it up without feeling like we're writing a lot of low-value tests.